.github/workflows/test.yml

```diff
@@ -26,7 +35,28 @@ jobs:
           python -m pip install --upgrade pip
           pip install pytest
           pip install .
+      - name: Set up aerie instance
+        run: |
+          docker compose -f docker-compose-test.yml up -d
+          docker images
+          docker ps -a
       - name: Test with pytest
         run: |
           cd tests
           pytest unit_tests
```

Comment:
The integration and unit tests can be two jobs. This'll help the tests run faster in the long-term, and make it easier to identify which tests fail in the short-term (since with this current structure, if unit tests fail, integration tests won't run)

Reply:
Thanks for the suggestion, I have now split up the unit tests and integration tests into two jobs in [e834d1f](https://github.com/NASA-AMMOS/aerie-cli/pull/54/commits/e834d1f59d7eb5ca76e3e9ad3587e39f0f98a384)